modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -2588,6 +2646,14 @@ private BytecodeBlock treatSourceForConstruct(
                 setField = setThisFieldCode(constructMtd, newValue, schemaFieldDef);
             }
 
+            if (isContainNameAnnotation(schemaField)) {
+                setField = new BytecodeBlock()
+                        .append(setField)
+                        .append(getThisFieldCode(constructMtd, schemaFieldDef))
```

Comment:
Why don't you use BytecodeExpressions.invoke...? It's more declarative and easier to understand

Reply:
I do not see an analogue of `BytecodeBlock#invokeVirtual(java.lang.reflect.Method)`, let's leave it as it is?